src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -101,7 +101,7 @@ public string SettingsPath
         /// Creates an instance of the AnalysisService class.
         /// </summary>
         /// <param name="settingsPath">Path to a PSScriptAnalyzer settings file.</param>
-        /// <param name="logger">An ILogger implementation used for writing log messages.</param>
+        /// <param name="logger">A logger used for writing log messages.</param>
```

Comment:
Might as well revert all these comment changes.  That removes ~25 more files from the PR.  :-)